adapters/primitive-adapters/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=primitive-adapters
```

Comment:
I'd kinda prefer the name `adapters-primitive` as it'd look better when importing dependencies (similar to the Retrofit adapters):

```groovy
implementation "com.squareup.sqldelight:adapters-primitive:1.4.4"
implementation "com.squareup.sqldelight:adapters-postgresql:1.4.4"
```

Compared to what we have now:

```groovy
implementation "com.squareup.sqldelight:primitive-adapters:1.4.4"
implementation "com.squareup.sqldelight:postgresql-adapters:1.4.4"
```

Compared to what we have now:

Reply:
whatever works, I think before 2.0 this would become something like "app.cash.sqldelight-sqlite:adapters"

Reply:
or if it stays in this repo "app.cash.sqldelight:sqlite-adapters", where we're doing target first since the target is how we're grouping similar libs